bazaarvoice/emodb
# Consolidated markdowns and posts

## Github Issue #

None

## What Are We Doing Here?

This PR consolidates redundant documentation in the markdowns directory and the github.io post.  It cases where both documents exist the github.io page is used and the old markdown documents have been updated to link to the new ones.  Additionally, in the few cases where the documents drifted the updates were applied to the github.io posts.

## How to Test and Verify

1. Check out this PR
2. Go to `docs` and run `start_local.sh`
3. Check out the posts and make sure the updated ones look good.
4. View each old markdown here and make sure it links to the correct post.

## Risk

Low, only documentation is modified.

### Level 
Low

### Required Testing

Smoke

### Risk Summary

No additional risk not already identified.

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
